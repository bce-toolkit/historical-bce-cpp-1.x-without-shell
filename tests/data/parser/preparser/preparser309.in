N2O3=NO2+NO