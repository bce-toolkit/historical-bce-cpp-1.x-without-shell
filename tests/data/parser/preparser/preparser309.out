N2O3 0 0
NO2 1 0
NO 1 0
